skyportal/handlers/photometry.py

```diff
@@ -68,6 +122,33 @@ def post(self):
             DBSession().add(p)
             DBSession().flush()
             ids.append(p.id)
+        if 'thumbnails' in data:
+            for thumb in data['thumbnails']:
+                basedir = Path(os.path.dirname(__file__))/'..'/'..'
+                if os.path.abspath(basedir).endswith('skyportal/skyportal'):
```

Comment:
What's happening here?

Reply:
On Travis, for some reason `Path(os.path.dirname(__file__))/'..'/'..'` returns something like `/home/user/skyportal/skyportal` instead of the expected `/home/user/skyportal` (which is what it returns locally), so I added this check